### ISSUE TITLE
Iframe content not painted at the right coordinate if the iframe is in a transformed element

### ISSUE THREAD

**Issue by paulrouget** [servo/servo#7377]
Related: https://github.com/servo/servo/issues/7282

```html
<body style="background:red">
  <div style="transform: scale(1)">
    <iframe src="http://example.com"></iframe>
  </div>
</body>
```

<img width="468" alt="screen shot 2015-08-26 at 10 51 15" src="https://cloud.githubusercontent.com/assets/373579/9489302/65757c9a-4be0-11e5-9150-74302bdd4ed7.png">

**Comment by paulrouget** [comment]
cc @pcwalton

**Comment by pcwalton** [comment]
Iiiiiiinteresting. Would have thought that would be fixed by #7282.

**Comment by paulrouget** [comment]
I will add to this that also the z-index is wrong:

```html
<style>

  body {
    background: yellow;
  }

  div, iframe {
    position: absolute;
    width: 200px;
    height: 200px;
  }

  iframe {
    border: 10px solid black;
    top: 0px;
    left: 0px;
  }

  div {
    left: 100px;
    top: 100px;
    background: red;
  }

</style>

<body>

  <iframe src='http://example.com'></iframe>
  <div> </div>
  
</body>
```

<img width="674" alt="screen shot 2015-09-29 at 10 43 26" src="https://cloud.githubusercontent.com/assets/373579/10159157/ef438d74-6696-11e5-97da-f404df03ab89.png">
